org.scala-ide.sdt.core.tests/src/org/scalaide/core/sbtbuilder/ScalaJavaDepWhenJavaIsWrongTest.scala

```diff
@@ -0,0 +1,83 @@
+package org.scalaide.core
+package sbtbuilder
+
+import org.eclipse.core.resources.IFile
+import org.eclipse.core.resources.IncrementalProjectBuilder
+import org.eclipse.core.runtime.NullProgressMonitor
+import org.junit.Assert._
+import org.junit.Before
+import org.junit.Test
+
+object ScalaJavaDepWhenJavaIsWrongTest extends testsetup.TestProjectSetup("scalajavadepjavaiswrong")
+
+class ScalaJavaDepWhenJavaIsWrongTest {
+  import ScalaJavaDepWhenJavaIsWrongTest._
+  import testsetup.SDTTestUtils._
+
+  @Before def setup: Unit = {
+    enableAutoBuild(false)
+  }
+
+  @Test def shouldCreateProjectFromWrongJava: Unit = {
```

Comment:
style points: side effecting methods should have `()` and methods without an equal sign are deprecated.